espanol/canudos.md

```diff
@@ -10,6 +10,8 @@ el Consejero:
 
 [Llegó el periodista miope con noticias importantes.](noticias/noticias.md)
 
+[Suddenly realized he was far from home and doesn't know how to speak Spanish.](../english/TheEpicSax/111_TheEpicSaxGuyArrives.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.